main.py

```diff
@@ -1,16 +1,9 @@
-#Required imports
-
-import tweepy #for twitter API
-import time   #for giving 15 seconds break between tweets
+import tweepy
```

Comment:
Imports in incorrect order, standard library comes before third party imports.

Reply:
Yep, a good catch. Implemented.